### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by Walidosgiu** [sleeyax/aternos-discord-bot#11] [General]
How can I customize the roles that can use the bot

**Comment by sleeyax** [comment]
It's not possible yet to specify which roles have access to which set of commands (unless you modify the source code ofc). At this time only the discord server admin can configure the bot and anyone else can use the other bot commands.

If you want to limit all commands to a specific role, you could set up permissions in a way that the bot can only read commands from a moderator text channel, for example.

**Comment by sleeyax** [comment]
Discord now provides a [built-in way](https://discord.com/blog/slash-commands-permissions-discord-apps-bots) to manage slash command permissions in your Discord server. There's no more need for an in-house solution!
